src/core/tests/Test_StatusHandlerTruncation.py

```diff
@@ -0,0 +1,784 @@
+        installation_msg = self.__get_message_json_from_substatus(complete_substatus_file_data, substatus_index=substatus_index)
+        installation_truncated_msg = self.__get_message_json_from_substatus(truncated_substatus_file_data, substatus_index=substatus_index)
+
+        self.assertEqual(installation_msg['installationActivityId'], installation_truncated_msg['installationActivityId'])
+        self.assertEqual(installation_msg['rebootStatus'], installation_truncated_msg['rebootStatus'])
+        self.assertEqual(installation_msg['maintenanceWindowExceeded'], installation_truncated_msg['maintenanceWindowExceeded'])
+        self.assertEqual(installation_msg['notSelectedPatchCount'], installation_truncated_msg['notSelectedPatchCount'])
+        self.assertEqual(installation_msg['excludedPatchCount'], installation_truncated_msg['excludedPatchCount'])
+        self.assertEqual(installation_msg['pendingPatchCount'], installation_truncated_msg['pendingPatchCount'])
+        self.assertEqual(installation_msg['installedPatchCount'], installation_truncated_msg['installedPatchCount'])
+        self.assertEqual(installation_msg['failedPatchCount'], installation_truncated_msg['failedPatchCount'])
+        self.assertEqual(installation_msg['startTime'], installation_truncated_msg['startTime'])
+        self.assertEqual(installation_msg['lastModifiedTime'], installation_truncated_msg['lastModifiedTime'])
+        self.assertEqual(installation_msg['maintenanceRunId'], installation_truncated_msg['maintenanceRunId'])
+
+    def __get_message_json_from_substatus(self, substatus_file_data, substatus_index=0):
+        return json.loads(substatus_file_data["status"]["substatus"][substatus_index]["formattedMessage"]["message"])
+
+    def __convert_performance_time_to_date_time_format(self, performance_time):
+        performance_time = abs(performance_time)
+
+        # Calc days, hours, minutes, and seconds
+        days, remainder = divmod(performance_time, 86400)  # 86400 seconds in a day
+        hours, remainder = divmod(remainder, 3600)  # 3600 seconds in an hour
+        minutes, seconds = divmod(remainder, 60)  # 60 seconds in a minute
+
+        # Format the result
+        formatted_time = "%d days, %d hours, %d minutes, %.6f seconds" % (int(days), int(hours), int(minutes), seconds)
+        return formatted_time
+
+    def __create_tmp_file_for_log_and_set_stdout(self):
+        """ Redirect sys console output to a tmp file directory for asserting log message output """
+        # Set up create tmp file for log and set sys.stdout to it
+        self.temp_stdout = tempfile.NamedTemporaryFile(delete=False, mode="w+")
+        # print('tmp file directory', self.temp_stdout.name)  # use for debugging get tmp file to see output message. DO NOT DELETE!
+        self.saved_stdout = sys.stdout  # Save the original stdout
+        sys.stdout = self.temp_stdout  # set it to the tmp file
+
+    def __remove_tmp_file_reset_stdout(self):
```

Comment:
Should not be in use and removed after recent changes

Reply:
logic is removed

done.